### PR TITLE
basic types for board + fix CI

### PR DESCRIPTION
**Pull request by laptou**
- created basic types to be used for representing game state
- fixed continuous integration

### CODE REVIEW

**Review by owenxr — APPROVED** [review]
Board, state, and tile to be changed in later PR.

main.yml pass checks

File: lib/tile.ml [hunk 1]
```diff
@@ -0,0 +1 @@
+type tile = char
```
Comment (kyralee22) [inline lib/tile.ml:1]: Add points field for type tile in addition to the occupying letter (This might be outside the scope for Beta)

Comment (laptou) [reply lib/tile.ml:1]: I think this is outside the scope of our alpha, which is why I ended up removing references to this type in #4

Comment (kyralee22) [reply lib/tile.ml:1]: Ah yeah okay my next question was wondering if we needed a curr_tile in game_state. sounds good

**Review by kyralee22 — APPROVED** [review]

**Pull request merged**